### PR TITLE
Add dummy property

### LINKED ISSUE #13: Export dummy property

### PR DESCRIPTION
**Pull request by TheKeymaster**
Fixes #13 

@georgms could you review this?
This issue was fixed, because i cannot proceed with tests at the moment.

### CODE REVIEW

File: DokuwikiXMLExport.php [hunk 1]
```diff
@@ -37,6 +38,18 @@ class DokuwikiXMLExport
...
+     * This value is the key for a dummy property.
+     * Hotfix workaround for a bug @ FINDOLOGIC
+     */
+    const PROPERTY_DUMMY_KEY = 'dummy';
+
+    /**
+     * This value is the value for a dummy property.
+     * Hotfix workaround for a bug @ FINDOLOGIC
+     */
+    const PROPERTY_DUMMY_VALUE = array('dummy');
```
Comment (georgms) [inline DokuwikiXMLExport.php:50]: Properties do not support multiple values so you can just use a single value here.

**Review by georgms — CHANGES_REQUESTED** [review]
LGTM

File: DokuwikiXMLExport.php (hunk 1 above)
Comment (TheKeymaster) [reply DokuwikiXMLExport.php:50]: @georgms 

`libflexport` only accepts an array here.
The reason for this is to support multible usergroups.
If I would set multible values e.g. `const PROPERTY_DUMMY_VALUE = array('dummy', 'dummy2');`
The result would look like this:
selection_097
If I wouldn't use an array here there wouldn't be any properties at all.

Please re-review this, thanks!

Comment (georgms) [reply DokuwikiXMLExport.php:50]: Got it, LGTM!

**Review by georgms — APPROVED** [review]

**Pull request merged**